### PR TITLE
Links and content update.

### PR DESCRIPTION
**Pull request by padmajabhol**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Updated the links, changed the pdf content, repositioned Github's logo.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by riteshsp2000** [comment]
@padmajabhol, i guess you didn't get the changes that we discussed,

1. Don't change the sponsor us link, that hasn't changed at all. Keep it as it is. (Revert the changes that you made for this)
2. In the Join us section, remove the part where the typeform pops up after clicking on Sponsor Us.
3. Instead of Sponsor Us, rename it to `Become a Community Partner`
4. Add a redirect to this button to open the link that I mentioned in the next tab.

File: src/components/SponsorsSection/Sponsor.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { SponsorContainer, HeadingSection, SponsorIcon, Wrapper } from './styles
 import { sponsors } from '../../../config/content';
 
 const links = {
-  sponsor: 'https://hacknitr21.typeform.com/to/maI5lJ4t',
+  sponsor: 'https://partners.hacknitr.tech/',
 };
```
Comment (riteshsp2000) [inline src/components/SponsorsSection/Sponsor.jsx:12]: - Revert this change to as it was.
- Use the config object from config/website.js file which contains all the links. 
- Add the community partner link here itself and use it.

File: src/components/SponsorsSection/Sponsor.jsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ function Sponsor() {
       id='sponsors'
       title={sponsors.title}
       description={sponsors.content}
-      link={{ text: 'Sponsor Us', link: links.sponsor }}
+      link={{ text: 'Become a community partner.', link: links.sponsor }}
     >
```
Comment (riteshsp2000) [inline src/components/SponsorsSection/Sponsor.jsx:70]: Revert this as well. The changes weren't to be done here. We have a dedicated sponsor us section now, check the live website ones (just below the hero section).

**Review by riteshsp2000 — CHANGES_REQUESTED** [review]

**Review by riteshsp2000 — APPROVED** [review]

**Review by abhibhaw — APPROVED** [review]
